### ISSUE TITLE
[Request future] autotized proxy

### ISSUE THREAD

**Issue by SergeyDjam** [qTox/qTox#2966]
Hello. Please, add to form settings autorized proxy

**Comment by ProMcTagonist** [comment]
Can you please explain with more details what you mean?

**Comment by SergeyDjam** [comment]
Now Playing missing fields to configure proxy authentication. There are times when you need to enter a user name and password for that would connect to the proxy to access the Internet

**Comment by ProMcTagonist** [comment]
https://github.com/tux3/qTox/issues/1286

**Comment by SergeyDjam** [comment]
Yes

**Comment by ProMcTagonist** [comment]
Blocked by https://github.com/irungentoo/toxcore/issues/1268

**Comment by zetok** [comment]
Since it's a duplicate of #1286, closing.
